Documentation/Signoffs/User-Interface-npatel45.md

```diff
@@ -0,0 +1,144 @@
+# Vending Machine Subsystem - User-Interface
+### Function of the Subsystem:
+---
+When designing vending machine the main component of the machine is to get input from the user and process that information and send that information to the rest of the parts corresponding to it. So, in the same way for ECE board vending machine the student must be able to input their information, and select products from given set of  choices so to target this requirement, UI (User Interface) subsystem is required.
+ 
+### Specifications & Constraints:
```

Comment:
What about size? Will this display actually allow the operator and users to read what is needed? Is there a good reason to use a very small display? Big displays give the impression of quality.